packages/react-core/src/components/Card/Card.tsx

```diff
@@ -79,6 +76,25 @@ export const Card: React.FunctionComponent<CardProps> = ({
     console.warn('Card: Cannot use isCompact with isLarge. Defaulting to isCompact');
     isLarge = false;
   }
+
+  if (isHoverable) {
+    // eslint-disable-next-line no-console
+    console.warn("Card: the 'isHoverable' prop is deprecated. Use isSelectable instead.");
```

Comment:
I don't thinks this is necessary.  We don't do this anywhere else that I can find.